ROADMAP.md

```diff
@@ -0,0 +1,46 @@
+# Roadmap
+
+## Towards seamless upgrades
+
+TBD
+
+## Pipeline
+
+### Support feature flags
+
+TBD
+
```

Comment:
Similar to `Pipeline` we can have section for `Triggers` also
As Triggers supports 
* custom interceptor and there is a plan to install builtin interceptor by admin(operator) cc @dibyom correct me if miss something
* Multi tenant EL support cc @khrm please add if anything we need to handle from Operator side

Reply:
> custom interceptor and there is a plan to install builtin interceptor by admin(operator) cc @dibyom correct me if miss something

right now the plan is to bundle the 4 interceptors as we do today but of course a downstream admin can decide to add/remove/change the set of interceptors they ship.